### DISCUSSION TITLE
kernel  watchdog: BUG: soft lockup

### DISCUSSION THREAD

**Discussion by 983183947** [k3s-io/k3s#8163] [Q&A]
I have encountered the following problems in different computer rooms. Now the server often freezes and even affects the host machine to freeze.
------------------------------------------------------------------------------------
Message from' syslogdck3s7 at Aug9 15:16:53 kernel
kernel:[ 4071.489871] watchdog: BUG: soft lockup. - CPU#7 stuck for 28s! -[gemu-ga:864]

**Comment by brandond** [comment]
This would be a hardware or kernel issue. User processes are not responsible for avoiding soft locks.
